### ISSUE TITLE
r restarting) on display after bed heats

### ISSUE THREAD

**Issue by G-Forge** [MarlinFirmware/Marlin#5894]
I have been running RC8 bf from December, all OK no problems.

Today I loaded current bugfix. Prints OK, no problem - except
after the bed heats up the 12864 displays **_r restarting)_** and then starts heating the extruder. Print starts OK when it gets to temp and there are no problems at all.

For some reason it looks like it displayed this string
```cpp
#define MSG_ERR_STOPPED "Printer stopped due to errors.
                         Fix the error and use M999 to restart.
                         (Temperature is reset. Set it after restarting)"
```
but continued on.

**Comment by thinkyhead** [comment]
That's pretty strange. It sounds like a buffer overrun or stack overflow because it's printing strings from random places. The "r restarting)" string comes from here:
```cpp
#define MSG_ERR_STOPPED "Printer stopped due to errors.
                         Fix the error and use M999 to restart.
                         (Temperature is reset. Set it after restarting)"
```
What version of Arduino did you use to do the build and upload?

**Comment by Blue-Marlin** [comment]
More intersting to me would be - what should be printed?
Looks to me as if it could be a progmem vs. ram -string problem.

**Comment by G-Forge** [comment]
The version of Arduino was 1.6.7 and the Marlin RC8 bugfix of 3/22/17.

**Comment by thinkyhead** [comment]
> More intersting to me would be - what should be printed?

Perhaps `MSG_HEATING_COMPLETE` ("Heating done.")

@G-Forge
- Do you have any `M117` commands in your starting GCode?
- Please test `RCBugFix` because this bug may already be gone.

**Comment by G-Forge** [comment]
>More intersting to me would be - what should be printed?
Perhaps MSG_HEATING_COMPLETE ("Heating done.")

Yes, I appears when _heating done_ would normally appear. I have no M117 commands.
I can't test again, OOO until a replacement RAMPS arrives.

**Comment by Blue-Marlin** [comment]
What is `const char* const message` in 
```
void lcd_setstatuspgm(const char* const message, uint8_t level) {
```
in ultralcd.cpp and why is it different from
```
void lcd_setstatuspgm(const char* message, const uint8_t level=0);
```
in ultralcd.h?

**Comment by benlye** [comment]
Not sure if it's the same issue, but I see a similar type of behaviour after updating to the latest RCBugFix.

When issuing M104 S180 to heat my extruder, my LCD (a variant of the RepRapDiscount Full Graphic Smart Controller) displays "LLED." in the status line.  M104 S0 takes it back to "3D Printer Ready".

**Comment by benlye** [comment]
M109 has the same effect - 'LLED.' on the screen until it reaches the specified temperature, then the message changes to 'Heating done.'

I suppose 'LLED.' might come from here:
```
#define MSG_KILLED                          "KILLED. "
```

**Comment by benlye** [comment]
Related to #5944 and #5876?

**Comment by thinkyhead** [comment]
Should be fixed by #5948.
